azure-proto-compute/AvailabilitySetOperations.cs

```diff
@@ -35,25 +58,52 @@ internal AvailabilitySetOperations(AzureResourceManagerClientOptions options, Re
 
+        /// <summary>
+        /// The operation to delete an availability set. 
+        /// </summary>
+        /// <returns> A response with the <see cref="ArmResponse{Response}"/> operation for this resource. </returns>
         public ArmResponse<Response> Delete()
         {
             return new ArmResponse(Operations.Delete(Id.ResourceGroup, Id.Name));
         }
 
+        /// <summary>
+        /// The operation to delete an availability set. 
+        /// </summary>
+        /// <param name="cancellationToken"> A token to allow the caller to cancel the call to the service. The default value is <see cref="P:System.Threading.CancellationToken.None" />. </param>
+        /// <returns> A <see cref="Task"/> that on completion returns a response with the <see cref="ArmResponse{Response}"/> operation for this resource. </returns>
         public async Task<ArmResponse<Response>> DeleteAsync(CancellationToken cancellationToken = default)
         {
             return new ArmResponse(await Operations.DeleteAsync(Id.ResourceGroup, Id.Name, cancellationToken));
         }
 
+        /// <summary>
+        /// The operation to delete an availability set. 
+        /// </summary>
+        /// <param name="cancellationToken"> A token to allow the caller to cancel the call to the service. The default value is <see cref="P:System.Threading.CancellationToken.None" />. </param>
+        /// <remarks>
+        /// <see href="https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-longrunning"> Details on long running operation object. </see>
+        /// </remarks>
+        /// <returns> An <see cref="ArmOperation{Response}"/> that allows polling for completion of the operation. </returns>
         public ArmOperation<Response> StartDelete(CancellationToken cancellationToken = default)
         {
             return new ArmVoidOperation(Operations.Delete(Id.ResourceGroup, Id.Name, cancellationToken));
         }
 
+        /// <summary>
+        /// The operation to delete an availability set. 
+        /// </summary>
+        /// <param name="cancellationToken"> A token to allow the caller to cancel the call to the service. The default value is <see cref="P:System.Threading.CancellationToken.None" />. </param>
+        /// <remarks>
+        /// <see href="https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-longrunning"> Details on long running operation object. </see>
+        /// </remarks>
+        /// <returns> A <see cref="Task"/> that on completion returns an <see cref="ArmOperation{Response}"/> that allows polling for completion of the operation. </returns>
         public async Task<ArmOperation<Response>> StartDeleteAsync(CancellationToken cancellationToken = default)
         {
             return new ArmVoidOperation(await Operations.DeleteAsync(Id.ResourceGroup, Id.Name, cancellationToken));
         }
+
+        /// <inheritdoc/>
```

Comment:
same comment on inherit it hides the return type since the base only calls out TOperations